vibe_code_apps_20/wattlab/studio/pages/fuel_dashboard.py

```
@@ -277,26 +268,75 @@ def _render_monthly(campus: Campus, summary: dict[str, Any], go) -> None:
+                f"Use all {choices['available_n']} overlapping months (not just full years)",
+                value=True,
+                key="fuel_dash_fit_all",
+            )
+        years_fit = st.slider(
+            "Fit window: last N years",
+            min_value=1,
+            max_value=max_y,
+            value=int(choices["default_years"]),
+            key="fuel_dash_fit_years",
+            disabled=use_all and choices["available_n"] > max_y * 12,
+            help="OLS gas×HDD / elec×CDD uses the latest N×12 months (or all overlap).",
```

Ruff (0.15.21)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)